mmv1/third_party/terraform/tests/resource_google_service_account_iam_test.go

```diff
@@ -110,6 +111,16 @@ func TestAccServiceAccountIamMember(t *testing.T) {
 				ImportState:       true,
 				ImportStateVerify: true,
 			},
+			{
+				Config:             testAccServiceAccountIamMember_basic(account, strings.ToUpper(email)),
+				PlanOnly:           true,
+				ExpectNonEmptyPlan: false,
```

Comment:
```suggestion
```

You can omit these, they're false by default. I keep doing double-takes as I scroll by because `ExpectNonEmptyPlan` is a signal a resource has a permadiff.

Reply:
ah, gotcha, yeah this was just a copy job from some of the other function. I'll just clean up the whole file

Reply:
wait no, this is definitely needed

we expect an empty plan so this needs to be set

Reply:
The default value of a boolean is false

Reply:
i am wrong, you are right